src/components/dialogs/commons/utils.tsx

```diff
@@ -5,10 +5,10 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-export function getIdOrValue(value: { id: string; label: string } | string) {
+export function getIdOrValue(value: any) {
     return typeof value !== 'string' ? value?.id ?? null : value;
 }
 
-export function getLabelOrValue(value: { id: string; label: string } | string) {
+export function getLabelOrValue(value: any) {
```

Comment:
why did you remove typescript check?

Reply:
Compilation errors, too hard to properly type, seen with @flomillot